nafis-vw-header/includes/css/scss/_mixins-fonts.scss

```
@@ -1,3 +1,19 @@
+@mixin primaryFontStackLight {
+	font-family: "VWHeadLight", arial, verdana, sans-serif;
```

All `font-family` fallbacks: Please add Helvetica instead of Verdana as one of the fallbacks and use Helvetica before Arial.
```suggestion
	font-family: "VWHeadLight", Helvetica, Arial, sans-serif;
```

---

Update all font mixins with suggested fallbacks: acaa80f